### PR TITLE
Add cc metadata file to the repository

### LINKED ISSUE #39: The repository does not contain a CC metadata file
## Description
Each repo in the CC organisation should have a `.cc-metadata.yml` file. This one does not.

## Expectation
The `.cc-metadata.yml` file should be present.

## Resolution

- [ ] I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by snehal199**
Fixes #39 by @dhruvkb

Adds .cc-metadata.yml file to the repository.

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [ ] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: .cc-metadata.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Whether this GitHub repo is engineering related
+engineering_project: true
+# Name of the repository/project in English
+english_name: cc-licenses
```
Comment (kgodey) [inline .cc-metadata.yml:4]: Please change to "CC Licenses"

File: .cc-metadata.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+# All technologies used
+technologies: Python, HTML
+# Whether this repository should be featured on the CC Open Source site
+featured: false
```
Comment (kgodey) [inline .cc-metadata.yml:8]: please add newline at the end of the file.

**Review by kgodey — CHANGES_REQUESTED** [review]

File: .cc-metadata.yml (hunk 2 above)
Comment (snehal199) [reply .cc-metadata.yml]: Made the suggested changes @kgodey

File: .cc-metadata.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Whether this GitHub repo is engineering related
+engineering_project: true
+# Name of the repository/project in English
+english_name: CC Licenses
+# All technologies used
+technologies: Python, HTML
+# Whether this repository should be featured on the CC Open Source site
+featured: false
```
Comment (Audiosutras) [inline .cc-metadata.yml:8]:
```suggestion
featured: false

```

Comment (snehal199) [reply .cc-metadata.yml:8]: Done @Audiosutras ! Sorry for the hiccup

**Comment by kgodey** [comment]
the tests are failing, could you investigate and fix, @snehal199?

**Comment by dpoirier** [comment]
> the tests are failing, could you investigate and fix, @snehal199?

FYI, It looks as if the tests failed due to no final newline on the new metadata file.

**Comment by snehal199** [comment]
Made the necessary change and the tests pass now @kgodey , @dpoirier

**Review by kgodey — APPROVED** [review]

**Pull request merged**